### PR TITLE
Update to 5.0.0

### PR DESCRIPTION
**Pull request by ocefpaf**
Bump to latest version.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,10 +25,8 @@ requirements:
     - python
     - pickleshare
     - simplegeneric >0.8
-# These are listed in the setup.py, but do not cause the build tests to fail.
-# In order to enable these dependencies, additional tests will need to be defined.
-#    - prompt_toolkit >=0.60
-#    - pygments
+    - prompt_toolkit >=1.0.3,<2.0.0
+    - pygments
```
Comment (ocefpaf) [inline recipe/meta.yaml]: These two are now needed to pass the import test. I am wondering if we should add all the extras listed in `setup.py` too.

```yaml
    # Extras
    - ipyparallel
    - qtconsole
    - ipykernel
    - nbformat
    - notebook
    - ipywidgets
    - nbconvert
    - pathlib2  # [py2k]
    - appnope  # [osx]
    - colorama  # [win]
    - win_unicode_console  # [win] >=0.5
```

Comment (minrk) [inline recipe/meta.yaml]: Most of those are only the extras that are legacies of the pre-Jupyter days. The uncondiational platform dependencies: pathlib2, appnope, colorama, and win_unicode_console should be included, though.

Comment (ocefpaf) [inline recipe/meta.yaml]: Done.

Comment (minrk) [inline recipe/meta.yaml]: Thanks! I'm guessing some of these are not packaged, yet.

Comment (ocefpaf) [inline recipe/meta.yaml]: I am working on `win_unicode_console` now.

**Comment by ocefpaf** [comment]
Windows is pending the feedstock creation at https://github.com/conda-forge/win_unicode_console-feedstock. But Travis-CI is not helping there :unamused: 

@minrk if you want to merge this to get the OS X and Linux up in the channel I am :+1: 

I can re-start AppVeyor as soon as `win_unicode_console` is uploaded to "fix" the Windows build.

**Comment by jakirkham** [comment]
👍 on whatever you guys decide.

Feel free to keep retrying restarting Travis CI at staged-recipes, @ocefpaf. I have found it does eventually clear, but not right away. It would be great to get to the bottom of the Travis CI API issues. I have opened a fresh issue with them upstream to see if we can get to the bottom of this.

@pelson and I have tried to add a few things to `conda-smithy` to make this work better they did help locally (though the API is still a bit flaky sadly), but they probably should go into a patch (i.e. 0.10.5) release to see if they might improve the situation without having to do a full on 1.0.0 release.

**Comment by minrk** [comment]
Still no update from appveyor, but merging here to get things started. Thanks!

**Comment by jakirkham** [comment]
:tada: `win-unicode-console` has finally cleared. :tada:

Currently, `win-unicode-console` is finishing its build. Restarting `master` for `ipython` on AppVeyor so we can get it built.

**Comment by jakirkham** [comment]
Windows packages for `ipython` version `5.0.0` are now deployed. 😄

**Comment by pelson** [comment]
This is pretty cool. Amazing turnaround @ocefpaf (I have screenshots to put in my presentation too)!  👍

**Pull request merged**